### PR TITLE
Update Set-Mailbox.md

### LINKED ISSUE #9953: MessageCopyForSMTPClientSubmissionEnabled on Set-Mailbox is empty
The -MessageCopyForSMTPClientSubmissionEnabled description is not filled out. Can someone fill out this description so we know what that command will do?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c7f835af-5905-368a-df68-6646782ad807
* Version Independent ID: 153ce2d2-dab5-9c8f-c511-ee3bc8ee2dbc
* Content: [Set-Mailbox (ExchangePowerShell)](https://learn.microsoft.com/en-us/powershell/module/exchange/set-mailbox?view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/Set-Mailbox.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/exchange-ps/exchange/Set-Mailbox.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

### PR DESCRIPTION
**Pull request by DMoenks**
Fixes #9953

### CODE REVIEW

File: exchange/exchange-ps/exchange/Set-Mailbox.md [hunk 1]
```diff
@@ -3794,7 +3794,10 @@ Accept wildcard characters: False
 ### -MessageCopyForSMTPClientSubmissionEnabled
 This parameter is available only in the cloud-based service.
 
-{{ Fill MessageCopyForSMTPClientSubmissionEnabled Description }}
+The MessageCopyForSMTPClientSubmissionEnabled parameter specifies whether to store copies of messages in the mailbox that are submitted via SMTP. Valid values are:
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-Mailbox.md:3797]:
```suggestion
The MessageCopyForSMTPClientSubmissionEnabled parameter specifies whether to store copies of messages in the mailbox that are submitted via SMTP. Valid values are:
```

File: exchange/exchange-ps/exchange/Set-Mailbox.md [hunk 2]
```diff
@@ -3794,7 +3794,10 @@ Accept wildcard characters: False
...
+
+- $true: When a message is submitted for the mailbox by using SMTP, a copy of the message is stored in the mailbox. This is the default value.
+- $false: When a message is submitted for the mailbox by using SMTP, a copy of the message isn't stored in the mailbox.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-Mailbox.md:3800]:
```suggestion
- $true: When a message is submitted to the mailbox by using SMTP, a copy of the message is stored in the mailbox. This is the default value.
- $false: When a message is submitted to the mailbox by using SMTP, a copy of the message isn't stored in the mailbox.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DMoenks, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrisda** [comment]
@DMoenks, sorry, I was going through my email in order, and the comments on Issue #9953 led me to do my own PR for the changes before I saw that you also did (perfectly fine) changes for the same issue.

So, closing to avoid merge conflicts

**Pull request closed without merging**